pkgs/os-specific/linux/open-isns/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, openssl, fetchurl }:
+let
+  pname = "open-isns-0.95";
```

Comment:
I suggest instead creating a `version` attribute and using it in both the `name` and in `src.url`, to make updating easier. Note that you'll need `rec` if you decide to move it into the attrset passed to `mkDerivation`.